Cqrs/Queries/Results/QuizDetailsQueryResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AspNetCoreVueStarter.Cqrs.Queries.Results
+{
+    public class QuizDetailsQueryResult
```

Comment:
Do not change property names. Rollback to previous names.)

Reply:
Done